pandas/_libs/tslibs/nattype.pyx

```diff
@@ -426,6 +426,7 @@ class NaTType(_NaT):
     toordinal = _make_error_func('toordinal', datetime)
     tzname = _make_error_func('tzname', datetime)
     utcoffset = _make_error_func('utcoffset', datetime)
+    fromisocalendar = _make_error_func('fromisocalendar', datetime)
```

Comment:
Does this need to be an in `if PY38` block? In other words, will there be a `Timestamp.fromisocalendar` for pandas 1.0 with python 3.7, or just py38 and above?